RELEASE.md

```diff
@@ -1,3 +1,27 @@
+# Release 0.16.0
+
+## Major Features
+* Added support for stream timeout in KafkaGroupIODataset
+* Renamed gstpufs to gsmemcachedfs
+* Added experimental KakfaBatchIODataset for online learning
+* Added gabor/laplacian/gaussian filter support
+* Added basic Arrow string type support for BigQuery reader
+* Added ElasticSearch dataset support
+* Added Prewitt and Sobel filter support
+* Added arbitary dimensional support for `decode_json`
+* Added bool data type support for `decode_json`
+
```

Comment:
@yongtang can we mention the tutorial updates and the availability of docker images as well?